Documentation/kubernetes-upgrade.md

```diff
@@ -0,0 +1,75 @@
+# Upgrading Kubernetes
+
+This document describes upgrading the Kubernetes components on a cluster's master and worker nodes. For general information on Kubernetes cluster management, upgrades (including more advanced topics such as major API version upgrades) see the [Kubernetes upstream documentation](http://kubernetes.io/v1.1/docs/admin/cluster-management.html) and [version upgrade notes](http://kubernetes.io/v1.1/docs/design/versioning.html#upgrades)
+
+**NOTE:** The following upgrade documentation is for installations based on the CoreOS + Kubernetes step-by-step [installation guide](https://coreos.com/kubernetes/docs/latest/getting-started.html). Upgrade documentation for the AWS cloud-formation based installation is forthcoming.
+
+## Upgrading the Kubelet
```

Comment:
I think it's worth noting in this section that it applies to both master and worker nodes. I suspect that some folks don't think of a master as being a "kubelet".